docs/developers/celestia-node-key.mdx

```diff
@@ -91,6 +91,14 @@ This will load the key `<key-name>` into the directory of the light node.
 </TabItem>
 </Tabs>
 
+
+Further flags you can use to customize your key are the following:
+
+* `--node.store`: Specifies a different directory you can use to
+  save your node data and configurations. Expects a path to a directory.
+* `--p2p.network`: Specifies which network you want the key for. Values
+  are `arabica` and `mocha`.
```

Comment:
We should mention somewhere in the docs that the node will only pick up keys that are inside the `node.store` directory under `/keys` so that users should point their `cel-key` utility at the correct directory via the `node.store` or `p2p.network` flags if they've specified a custom directory or network other than arabica.

Reply:
resolved in new paragraph below